### PR TITLE
Adding cifkit to conda-forge (Python)

### PR DESCRIPTION
**Pull request by bobleesj**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by bobleesj** [comment]
@conda-forge/help-python Hello Team, ready for review! It is my first time submitting this. I hope to learn from your feedback.

**Comment by bobleesj** [comment]
@conda-forge/help-python Hi Team, it appears the CI tests are passing. I would appreciate if anyone could take a look.

File: recipes/cifkit/meta.yaml [hunk 1]
```diff
@@ -32,21 +32,21 @@ requirements:
...
-    - cifkit
+    - {{ name }}
   commands:
     - pip check
   requires:
     - pip
 
 about:
-  home: https://github.com/bobleesj/cifkit
+  home: https://github.com/bobleesj/{{ name }}
```
Comment (ocefpaf) [inline recipes/cifkit/meta.yaml:42]: Please don't use jinja variable here. It doesn't help maintain the recipe but it does obscure it and render the URL unclickable.

Comment (bobleesj) [reply recipes/cifkit/meta.yaml]: @ocefpaf  Thank you! No variables are used for any of the URLs.

Please review!

File: recipes/cifkit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+requirements:
+  host:
+    - python >=3.10
+    - setuptools
+    - pip
+
+  run:
+    - python >=3.10
+    - gemmi
+    - matplotlib
```
Comment (ocefpaf) [inline recipes/cifkit/meta.yaml:26]: Unless you need the qt backend explicitly, this is recommended for a lighter package.
```suggestion
    - matplotlib-base
```

File: recipes/cifkit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    - numpy
+    - pyvista
+    - scipy
+    - setuptools
+    - click
+
+test:
+  imports:
+    - {{ name }}
```
Comment (ocefpaf) [inline recipes/cifkit/meta.yaml:35]:
```suggestion
    - cifkit
```

**Comment by bobleesj** [comment]
@conda-forge/help-python @ocefpaf Thank you! Reflected all the comments.

**Pull request merged**